### PR TITLE
fix: expose arguments of embed() in the evaluator

### PR DESCRIPTION
**Pull request by gmastrapas**

### CODE REVIEW

File: finetuner/tuner/evaluation.py [hunk 1]
```diff
@@ -49,6 +53,16 @@ def __init__(
             match.
         :param catalog: A sequence of documents, against which the eval docs will be matched.
         :param embed_model: The embedding model to use, in order to extract document representations.
+        :param device: the computational device for `embed_model`, can be either
+            `cpu` or `cuda`.
+        :param batch_size: number of Documents in a batch for embedding
```
Comment (JoanFM) [inline finetuner/tuner/evaluation.py:58]: why is `batch_size` not a parameter of `embed`? why is it in the `Evaluation` constructor?

Comment (gmastrapas) [reply finetuner/tuner/evaluation.py:58]: It is a parameter of `embed`. The evaluator uses the embed method internally and I need a way to configure the embed behavior

Comment (JoanFM) [reply finetuner/tuner/evaluation.py:58]: My point is, u are telling with this design, that batch_size is part of the identity of an Evaluator. Meaning that if I want to evaluate with different batch sizes, this should change, while batch_size is just an operational parameter, but should not be vital and core to the Evaluator

Comment (gmastrapas) [reply finetuner/tuner/evaluation.py:58]: Ah I see, so you mean it should be  an argument of `evaluate`?

File: finetuner/tuner/evaluation.py [hunk 2]
```diff
@@ -38,6 +38,10 @@ def __init__(
         eval_data: 'DocumentSequence',
         catalog: 'DocumentSequence',
         embed_model: Optional['AnyDNN'] = None,
+        device: str = 'cpu',
+        batch_size: int = 256,
+        preprocess_fn: Optional['PreprocFnType'] = None,
+        collate_fn: Optional['CollateFnType'] = None,
         distance: str = 'cosine',
         limit: int = 20,
```
Comment (maximilianwerk) [inline finetuner/tuner/evaluation.py:46]: limit should be right after `embed_model`. Reasoning: `limit` is a logical parameter, while the others are "technical".

**Pull request merged**